bin/saya/scripts/2_world_config.sh

```
@@ -0,0 +1,40 @@
+# Set environment variables for the world configuration
+DOJO_WORLD_ADDRESS=         # Address of the Dojo World contract
+STARKNET_RPC_URL=           # Starknet RPC URL to interact with the network
+DOJO_PRIVATE_KEY=           # Private key for executing transactions
+DOJO_ACCOUNT_ADDRESS=       # Account address used for signing transactions
+
+# Set the differ program hash in the Dojo World contract
+cargo run -r --bin sozo -- \
+    execute $DOJO_WORLD_ADDRESS set_differ_program_hash \
+    -c 0xa73dd9546f9858577f9fdbe43fd629b6f12dc638652e11b6e29155f4c6328 \
+    --manifest-path examples/spawn-and-move/Scarb.toml \  # Path to Scarb project manifest
+    --rpc-url $STARKNET_RPC_URL \                         # Starknet RPC URL
```

Shellcheck

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)